community/sway/usr/share/sway/scripts/lock.sh

```diff
@@ -5,5 +5,5 @@ if [ -x "$(command -v gtklock)" ]; then
 elif [ -x "$(command -v waylock)" ]; then
     waylock -fork-on-lock
 elif [ -x "$(command -v swaylock)" ]; then
-    swaylock --daemonize --show-failed-attempts
-fi
\ No newline at end of file
+    swaylock --daemonize --show-failed-attempts --screenshots --clock --indicator --effect-blur 7x5 --effect-vignette 0.5:0.5 --fade-in 0.2
```

Comment:
Did you test this with the regular `swaylock`, not `swaylock-effects`? If so we can gladly add it.

Reply:
With test I mean: did it _not_ break?